pandas/tests/io/test_html.py

```diff
@@ -894,24 +851,23 @@ def test_parse_dates_combine(self):
         newdf = DataFrame({"datetime": raw_dates})
         tm.assert_frame_equal(newdf, res[0])
 
-    def test_computer_sales_page(self, datapath):
-        data = datapath("io", "data", "html", "computer_sales_page.html")
-        msg = (
-            r"Passed header=\[0,1\] are too many "
```

Comment:
Do we have a way of replacing this test?

Reply:
The already existing test_parser_error_on_empty_header_row is the same message as the first part (but I'm not sure if it's the exact same code path).

The new test_wikipedia_states_multiindex is intended as a "can we read a MultiIndex table?" to replace the second part.

Reply:
Ah OK. I would say `test_multiindex_header` replaces the second part then